src/output-json-flow.c

```diff
@@ -52,6 +52,104 @@
 #include "flow-storage.h"
 #include "util-exception-policy.h"
 
+#define FLOW_LOG_EXCEPTION_POLICY BIT_U8(0)
+
+typedef struct LogFlowCtx_ {
+    uint8_t flags;
```

Comment:
Should we just use a bool if there's just one flag? If the plan is to add more to this, this is good

Reply:
Will do, don't know if we'll want to add other optional fields in the future.